examples/jit/bernstein_vazirani.ll

```diff
@@ -0,0 +1,536 @@
+; ModuleID = 'Bernstein-Vazirani'
```

Comment:
I would suggest to check this in as bitcode instead, and make sure it contains the necessary qirlib pieces in it if needed.

Reply:
Done.